lab_3_generate_by_ngrams/main.py

```diff
@@ -41,6 +44,19 @@ def _tokenize(self, text: str) -> Optional[tuple[str, ...]]:
         In case of corrupt input arguments, None is returned.
         In case any of methods used return None, None is returned.
         """
+        if not isinstance(text, str) or text == '':
+            return None
+        clear_text = ''
+        for symbol in text.lower():
+            if not symbol.isalpha:
+                symbol = self.end_of_word_token
+            clear_text += symbol
+        if self.end_of_word_token * 2 in clear_text:
```

Comment:
Is it possible there are three EoW on a row? 4 EoW in a row? And so on...

Reply:
Fixed